python/mrc/_pymrc/include/pymrc/port_builders.hpp

```diff
@@ -157,8 +158,8 @@ struct PortBuilderUtil
 
         if (!mrc::node::PortRegistry::has_port_util(type_idx))
         {
-            VLOG(2) << "Registering PyMRC port util for: " << type_name<port_type_t>() << " "
-                    << "=> " << type_name<port_dtype_t>() << " " << type_idx.hash_code();
+            // VLOG(2) << "Registering PyMRC port util for: " << type_name<port_type_t>() << " "
```

Comment:
Would prefer this to be uncommented and put at a higher VLOG level if its too verbose; its very helpful in tracking down issues where we get a PyObject edge instead of an optimized edge.

Reply:
I'll test out higher VLOG levels. This code gets called before the logging can get initialized and generates a warning from glog stating that logging was performed before logging was set up.